### ISSUE TITLE
Stunner Editors - Loading unsaved assets in VS Code leads to an empty file

### ISSUE THREAD

**Issue by yesamer** [apache/incubator-kie-issues#70]
Reported here: https://github.com/kiegroup/kie-tools/issues/1210

The issue reproduces on **VSCode Windows only**.

**Steps to reproduce the issue**
1. Open _any_ BPMN or DMN assets in VSCode
2. Apply _any_ change to the asset, _do not_ save it
3. Close the VSCode Window. Despite the `isDirty` icon is visible, no confirmation dialog is shown
4. Reopen VSCode - The editor should try to reload the previously modified asset
5. The assets is totally empty

A video of the issue is available [here](https://www.youtube.com/watch?v=GB2XgW8F1G0)

**Comment by yesamer** [comment]
**Analysis result**
When a user closes VSCode with unsaved assets, there's a mechanism in place that automatically backup all the changes applied by the user in a hidden directory in the [user-space storage](https://superuser.com/questions/1225368/visual-studio-code-unsaved-files-location). At the editor opening, VSCode tries to restore the unsaved assets from that backup. For some reason, it fails and as a result, an emptied asset is shown. 

In the logs, we noticed this error:
`
DmnEditorEnvelopeApp.js:24181 TypeError: Cannot read properties of undefined (reading 'getContent')
    at __webpack_modules__.4678.KogitoEditorEnvelopeApiImpl.kogitoEditor_contentRequest (DmnEditorEnvelopeApp.js:23614:28)
    at __webpack_modules__.927.EnvelopeBusMessageManager.receive (DmnEditorEnvelopeApp.js:24178:52)
    at Object.receive (DmnEditorEnvelopeApp.js:24028:27)
    at __webpack_modules__.6729.EnvelopeClient.receive (DmnEditorEnvelopeApp.js:24354:33)
    at eventListener (DmnEditorEnvelopeApp.js:24333:62)
__webpack_modules__.927.EnvelopeBusMessageManager.receive @ DmnEditorEnvelopeApp.js:24181
`
Not sure if is related or not to the root cause, mostly because I noticed the same error on macos, where the issue doesn't occur.
